### PR TITLE
Bump sphinx to 4.5.0

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [ ] Code changes are covered by tests
- [ ] Code changes have been evaluated for compatibility/integration with TrendAnalysis
- [ ] New functions added to `__init__.py`
- [ ] API.rst is up to date, along with other sphinx docs pages
- [ ] Example notebooks are rerun and differences in results scrutinized
- [ ] Updated changelog

Fixes an incompatibility with jinja2.  Same as https://github.com/pvlib/pvlib-python/pull/1435.

### CODE REVIEW

**Comment by kanderso-nrel** [comment]
The RTD PR build looks good to me, but would be good to get another set of eyes before merging

**Review by mdeceglie — APPROVED** [review]
Looks good, thanks @kanderso-nrel!

**Pull request merged**
